src/codewhisperer/service/recommendationHandler.ts

```diff
@@ -131,7 +131,7 @@ export class RecommendationHandler {
                 CodeWhispererConstants.promiseTimeoutLimit * 1000
             )
         } catch (error) {
-            throw new Error(`${error instanceof Error ? error.message : error}`)
+            throw error
```

Comment:
since we don't have any specific handling in the catch block, should we simply remove the entire try..catch block here ? @andrewyuq @leigaol @zixlin7

Reply:
the error is already handled at `getRecommendations()` https://github.com/aws/aws-toolkit-vscode/blob/master/src/codewhisperer/service/recommendationHandler.ts#L229

Reply:
yes we can remove the try catch here